docs/api/helper/file-queue.md

```diff
@@ -26,11 +26,14 @@ If a name is not provided, the default queue will be used.
 
 ## Arguments
 
-| Argument        | Description                       | Type                         | Default value |
-| --------------- | --------------------------------- | ---------------------------- | ------------- |
-| `name`          | Name of queue to find or create.  | `string`                     |               |
-| `onFileAdded`   | Called when each file is added.   | `(file: UploadFile) => void` |               |
-| `onFileRemoved` | Called when each file is removed. | `(file: UploadFile) => void` |               |
+| Argument            | Description                               | Type                                             | Default value |
+| ------------------- | ----------------------------------------- | ------------------------------------------------ | ------------- |
+| `name`              | Name of queue to find or create.          | `string`                                         |               |
+| `onFileAdded`       | Called when each file is added.           | `(file: UploadFile) => void`                     |               |
+| `onFileRemoved`     | Called when each file is removed.         | `(file: UploadFile) => void`                     |               |
+| `onUploadStarted`   | Called when each upload begins.           | `(file: UploadFile) => void`                     |               |
+| `onUploadSucceeded` | Called when each upload request succeeds. | `(file: UploadFile, response: Response) => void` |               |
+| `onUploadFailed`    | Called when each upload request fails.    | `(file: UploadFile, response: Response) => void` |               |
```

Comment:
I find "each file" and "each upload" a little bit confusing wording. Maybe:

```suggestion
| `onFileAdded`       | Called when a file is added to this queue.           | `(file: UploadFile) => void`                     |               |
| `onFileRemoved`     | Called when a file is removed from this queue.         | `(file: UploadFile) => void`                     |               |
| `onUploadStarted`   | Called when upload starts for a file of this queue.           | `(file: UploadFile) => void`                     |               |
| `onUploadSucceeded` | Called when upload of a file of this queue succeeds. | `(file: UploadFile, response: Response) => void` |               |
| `onUploadFailed`    | Called when upload of a file of this queue fails fails.    | `(file: UploadFile, response: Response) => void` |               |
```

Reply:
Thanks! I committed this with a few grammatical adjustments.